lib/vauth/spnego_sspi.c

```diff
@@ -224,6 +225,8 @@ CURLcode Curl_auth_decode_spnego_message(struct Curl_easy *data,
   free(chlg);
 
   if(GSS_ERROR(nego->status)) {
+    failf(data, "InitializeSecurityContext failed: %s",
+          Curl_sspi_strerror(data->easy_conn, nego->status));
     return CURLE_OUT_OF_MEMORY;
```

Comment:
I'm with @jay, `CURLE_RECV_ERROR` might be slightly better. But none of the is ideal here...